templates/index.html

```diff
@@ -9,250 +9,266 @@
 					</li>
-					<li class="inline-logos__item">
-						<img class="inline-logos__image" src="{{ ASSET_SERVER_URL }}df4150e2-nec-logo.svg" alt="NEC logo" />
+					<li class="p-inline-images__item">
+						<img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}df4150e2-nec-logo.svg" alt="NEC logo" />
 					</li>
-					<li class="inline-logos__item">
-						<img class="inline-logos__image" src="{{ ASSET_SERVER_URL }}040783dd-verizon-logo.png" alt="Verizon logo" />
+					<li class="p-inline-images__item">
+						<img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}040783dd-verizon-logo.png" alt="Verizon logo" />
 					</li>
-					<li class="inline-logos__item">
-						<img class="inline-logos__image" src="{{ ASSET_SERVER_URL }}0f6861b9-oil-logo.png" alt="OpenStack Interoperability Lab">
+					<li class="p-inline-images__item">
+						<img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}0f6861b9-oil-logo.png" alt="OpenStack Interoperability Lab">
 					</li>
-					<li class="inline-logos__item">
-						<img class="inline-logos__image" src="{{ ASSET_SERVER_URL }}3c7e83fc-atandt-logo.svg" alt="AT&amp;T logo" />
+					<li class="p-inline-images__item">
+						<img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}3c7e83fc-atandt-logo.svg" alt="AT&amp;T logo" />
 					</li>
-					<li class="inline-logos__item">
-						<img class="inline-logos__image" src="{{ ASSET_SERVER_URL }}a8e8aa9e-deutschetelekom-logo.svg" alt="Deutche Telekom logo" />
+					<li class="p-inline-images__item">
+						<img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}a8e8aa9e-deutschetelekom-logo.svg" alt="Deutche Telekom logo" />
 					</li>
-					<li class="inline-logos__item">
-						<img class="inline-logos__image" src="{{ ASSET_SERVER_URL }}73135672-ntt-logo.svg" width="110" alt="NTT logo" />
+					<li class="p-inline-images__item">
+						<img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}73135672-ntt-logo.svg" alt="NTT logo" />
 					</li>
 				</ul>
 			</div>
 		</div>
 	</section>
-	<section class="row strip-white pricing">
-		<div class="inner-wrapper">
-			<div class="twelve-col">
+
+	<section class="p-strip-x-light pricing is-deep">
+		<div class="row">
+			<div class="col-8">
 				<h2>Pricing</h2>
-				<p class="eight-col">MAAS is supported by Canonical under the <a href="http://www.ubuntu.com/management/ubuntu-advantage" class="external">Ubuntu Advantage</a> service program. Ubuntu Advantage MAAS includes tools that enable deployment of a wider range of operating systems, SLAs on support and high availability reference architectures.</p>
+				<p>MAAS is supported by Canonical under the <a href="http://www.ubuntu.com/management/ubuntu-advantage" class="p-link--external">Ubuntu Advantage</a> service program. Ubuntu Advantage MAAS includes tools that enable deployment of a wider range of operating systems, SLAs on support and high availability reference architectures.</p>
```

Comment:
Requires an ```aria-label``` to explain where the link takes you too. Also explain that this is an external link.